packages/crawler/src/page-processors/simulator-page-processor.ts

```diff
@@ -1,39 +1,42 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+
 import Apify from 'apify';
-import { Logger } from 'logger';
+import { inject, injectable } from 'inversify';
+import { GlobalLogger } from 'logger';
 import { AccessibilityScanOperation } from '../page-operations/accessibility-scan-operation';
 import { ClickElementOperation } from '../page-operations/click-element-operation';
 import { EnqueueActiveElementsOperation } from '../page-operations/enqueue-active-elements-operation';
 import { Operation } from '../page-operations/operation';
 import { LocalBlobStore } from '../storage/local-blob-store';
 import { LocalDataStore } from '../storage/local-data-store';
-import { BlobStore, DataStore, scanResultStorageName } from '../storage/store-types';
+import { BlobStore, DataStore } from '../storage/store-types';
 import { ActiveElement } from '../utility/active-elements-finder';
 import { PageProcessorBase } from './page-processor-base';
 
+@injectable()
 // tslint:disable: no-unsafe-any
 export class SimulatorPageProcessor extends PageProcessorBase {
     public constructor(
-        protected readonly logger: Logger,
+        @inject(EnqueueActiveElementsOperation) protected readonly enqueueActiveElementsOp: EnqueueActiveElementsOperation,
+        @inject(ClickElementOperation) protected readonly clickElementOp: ClickElementOperation,
+        @inject(AccessibilityScanOperation) protected readonly accessibilityScanOp: AccessibilityScanOperation,
+        @inject(LocalDataStore) protected readonly dataStore: DataStore,
+        @inject(LocalBlobStore) protected readonly blobStore: BlobStore,
+        @inject(GlobalLogger) private readonly loggerSimulator: GlobalLogger,
```

Comment:
may just `logger` instead to avoid confusion about special logger functionality

Reply:
logger isn't injectable

Reply:
also I'll add a task to create a console logger so it doesn't need to take service configuration and print this unnecessary information

Reply:
I mean just to rename to logger instead

Reply:
if I use the same name, I'll get error 
"Class 'SimulatorPageProcessor' incorrectly extends base class 'PageProcessorBase'.
  Property 'logger' is private in type 'SimulatorPageProcessor' but not in type 'PageProcessorBase'."

same for enqueueLinksSimulator and gotoExtendedSimulator

Reply:
to resolve this just make same var scope in both base and derived classes like private or protected